launchToolbar/js/launchToolbar_meta.user.js

```diff
@@ -3,14 +3,14 @@
 // @namespace www.cobaltgroup.com
 // @include https://cdk.my.salesforce.com/a1E*
 // @include https://cdk--c.na57.visual.force.com/*
-// @version 4.8.2
+// @version 4.8.3
```

Comment:
Please update the @name --> Salesforce Launch Toolbar - Pre-Release.
Thank you!